101-vm-full-disk-encrypted-rhel/azuredeploy.json

```diff
@@ -0,0 +1,164 @@
+      }
+    },
+    "_artifactsLocationSasToken": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+      }
+    }
+  },
+  "variables": {
+    "createVmUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-vm-simple-rhel/azuredeploy.json",
+    "createVmDeploymentName": "[concat(uniquestring(parameters('vmName')), 'createVm')]",
+    "encryptVmUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-encrypt-running-linux-vm/azuredeploy.json",
+    "encryptVmDeploymentName": "[concat(uniquestring(parameters('vmName')), 'encryptVm')]",
+    "scriptsFolder": "scripts",
+    "scriptFileName": "setup_raid.sh",
+    "setupRaidUrl": "[concat(parameters('_artifactsLocation'), '/', variables('scriptsFolder'), '/', variables('scriptFileName'), parameters('_artifactsLocationSasToken'))]",
+    "setupRaidExtensionName": "setupRaidCustomScript",
+    "diskFormatQuery": "{\"dev_path\":\"/dev/md0\",\"name\":\"encryptedraid\",\"file_system\":\"ext4\"}"
+  },
+  "resources": [
+    {
+      "apiVersion": "2016-09-01",
+      "name": "[variables('createVmDeploymentName')]",
+      "type": "Microsoft.Resources/deployments",
+      "properties": {
+        "mode": "Incremental",
+        "parameters": {
+          "adminUsername": {
+            "value": "[parameters('adminUsername')]"
+          },
+          "adminPassword": {
+            "value": "[parameters('adminPassword')]"
+          },
+          "rhelOSVersion": {
+            "value": "7.2"
+          },
+          "vmName": {
+            "value": "[parameters('vmName')]"
+          }
+        },
+        "templateLink": {
+          "contentVersion": "1.0.0.0",
+          "uri": "[variables('createVmUrl')]"
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Compute/virtualMachines/extensions",
+      "name": "[concat(parameters('vmName'), '/', variables('setupRaidExtensionName'))]",
+      "apiVersion": "2016-03-30",
+      "location": "[resourceGroup().location]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Resources/deployments', variables('createVmDeploymentName'))]"
+      ],
+      "properties": {
+        "publisher": "Microsoft.OSTCExtensions",
+        "type": "CustomScriptForLinux",
+        "typeHandlerVersion": "1.5",
+        "autoUpgradeMinorVersion": true,
+        "settings": {
+          "fileUris": [
+            "[variables('setupRaidUrl')]"
+          ],
+          "commandToExecute": "./setup_raid.sh"
+        },
+        "protectedSettings": {}
+      }
+    },
+    {
+      "apiVersion": "2016-09-01",
```

Comment:
The latest apiVersion for "Microsoft.Resources/deployments" is "2016-02-01". Please change this apiVersion to that.